orchestra/project_api/api.py

```diff
@@ -71,13 +75,16 @@ def get_workflow_steps(workflow_slug, version_slug):
     workflow_version = WorkflowVersion.objects.get(
         slug=version_slug,
         workflow__slug=workflow_slug)
+    steps = Step.objects.prefetch_related(
```

Comment:
very cool! do you need the prefetch_related? i'd imagine that comes for free with the Step